pkg/api/authn.go

```diff
@@ -55,7 +55,18 @@ func bearerAuthHandler(ctlr *Controller) mux.MiddlewareFunc {
 			}
 			vars := mux.Vars(request)
 			name := vars["name"]
+
```

Comment:
Why do we need this?
The request reaches this code if the client already has a bearer token, and if he already knows it needs to use a bearer token, it doesn't need the mgmt api?

Reply:
anonymous case?

Reply:
hmm I think you have a point, although this code is reached without token first
then it will authFail(response, permissions.WWWAuthenticateHeader, 0) with bearer headers
then client will get the token using bearer headers
then it will try the same request but with token this time.

But yeah, if UI makes a /v2/ GET request it can tell by the WWWAuthenticateHeader that a bearer auth is in place.

Reply:
Do we have a use case with bearer auth for the UI?

Reply:
Ok, let's fix this.